pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -821,6 +826,77 @@ func (r *ReconcileHyperConverged) ensureKubeVirtTemplateValidator(instance *hcov
 	return r.client.Status().Update(context.TODO(), instance)
 }
 
+func newKubeVirtStorageConfigForCR(cr *hcov1alpha1.HyperConverged, namespace string) *corev1.ConfigMap {
+	var volumeMode string
+
+	if bareMetalPlatform {
+		volumeMode = "Block"
+	} else {
+		volumeMode = "Filesystem"
+	}
+
+	labels := map[string]string{
+		"app": cr.Name,
+	}
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kubevirt-config-storage-class-defaults",
+			Labels:    labels,
+			Namespace: namespace,
+		},
+		Data: map[string]string{
+			"accessMode": "ReadWriteMany",
+			"volumeMode": volumeMode,
+		},
+	}
+}
+
+func (r *ReconcileHyperConverged) isBareMetalPlatform(request reconcile.Request) bool {
+	infra := &apiconfigv1.Infrastructure{}
```

Comment:
I would rather the user set a RHHI or Baremetal flag instead of us auto detecting.   You can add the flag [here](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/pkg/apis/hco/v1alpha1/hyperconverged_types.go#L19).

Reply:
done